test/.eslintrc

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
Is this just for IDEs to pick up? (because the builder archetype scripts already handle linting of test code).